lambda/main.go

```diff
@@ -162,6 +162,7 @@ func isPreRelease(version string) bool {
 	terms := []string{
 		"pre",
 		"beta",
+		"alpha",
```

Comment:
Curious if we should just use one of the semver libraries instead, like https://github.com/blang/semver?